File: 백준/Bronze/15025. Judging Moose/Judging Moose.py
import sys
input=lambda:sys.stdin.readline().rstrip()
l,r=map(int,input().split())
if l==0 and r==0:
    print("Not a moose")
else:
    if l==r:
        print("Even",l*2)
    else:
        print("Odd", l*2 if l>r else r*2)

File: 백준/Bronze/15025. Judging Moose/Judging Moose.c
#include <stdio.h>
int main(void) {
    int l, r;
    scanf("%d %d", &l, &r);
    if (l == 0 && r == 0) {
        printf("%s", "Not a moose");
    } else {
        if (l == r)
            printf("Even %d", l * 2);
        else
            printf("Odd %d", l > r ? l * 2 : r * 2);
    }
    return 0;
}